### PR TITLE
Changed KeyckloakTextInput to TextControl in NewRealmForm

### LINKED ISSUE #27595: Change KeyckloakTextInput to TextControl in NewRealmForm
### Description

Change KeyckloakTextInput to TextControl in NewRealmForm

### PR DESCRIPTION
**Pull request by agagancarczyk**
Closes https://github.com/keycloak/keycloak/issues/27595

### CODE REVIEW

File: js/apps/admin-ui/src/realm/add/NewRealmForm.tsx [hunk 1]
```diff
@@ -64,60 +62,51 @@ export default function NewRealmForm() {
...
+                    name="enabled"
+                    label={t("on")}
+                    labelOff={t("off")}
+                    isChecked={field.value}
+                    onChange={field.onChange}
+                    aria-label={t("enabled")}
+                  />
+                )}
+              />
+            </FormGroup>
```
Comment (jonkoops) [inline js/apps/admin-ui/src/realm/add/NewRealmForm.tsx:99]: We could replace this with a `SwitchControl` while we're at it.

**Review by jonkoops — DISMISSED** [review]
LGTM, but perhaps we can convert the whole form over to controls.

File: js/apps/admin-ui/src/realm/add/NewRealmForm.tsx (hunk 1 above)
Comment (edewit) [inline js/apps/admin-ui/src/realm/add/NewRealmForm.tsx:99]: we could replace this with a DefaultSwitchControl

**Review by edewit — CHANGES_REQUESTED** [review]

File: js/apps/admin-ui/src/realm/add/NewRealmForm.tsx [hunk 2]
```diff
@@ -29,14 +24,17 @@ export default function NewRealmForm() {
   const { refresh: refreshRealms } = useRealms();
   const { addAlert, addError } = useAlerts();
   const [realm, setRealm] = useState<RealmRepresentation>();
+  const [defaultValue, setDefaultValue] = useState(true);
```
Comment (jonkoops) [inline js/apps/admin-ui/src/realm/add/NewRealmForm.tsx:27]: Why do we now need this additional state and effect? If you need to pass a default value you can do the following:

```
<DefaultSwitchControl defaultValue={true} />
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

File: js/apps/admin-ui/src/realm/add/NewRealmForm.tsx (hunk 2 above)
Comment (agagancarczyk) [reply js/apps/admin-ui/src/realm/add/NewRealmForm.tsx:27]: It doesn't work, I tried to modify DefaultSwitchControl component by adding defaultValue as a prop but it still complained that SwitchProps doesn't have defaultValue

Comment (jonkoops) [reply js/apps/admin-ui/src/realm/add/NewRealmForm.tsx:27]: It should, we're also using `defaultValue` in [other places.](https://github.com/keycloak/keycloak/blob/4b697009d3336dbb702eecc83083020cc16b5de3/js/apps/admin-ui/src/client-scopes/details/ScopeForm.tsx#L160)

Comment (agagancarczyk) [reply js/apps/admin-ui/src/realm/add/NewRealmForm.tsx]: This should be fixed now

**Review by jonkoops — DISMISSED** [review]

**Review by edewit — DISMISSED** [review]

File: js/apps/admin-ui/cypress/support/pages/admin-ui/CreateRealmPage.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 export default class CreateRealmPage {
   #clearBtn = ".pf-c-file-upload__file-select button:last-child";
   #modalClearBtn = "clear-button";
-  #realmNameInput = "#kc-realm-name";
+  #realmNameInput = "#realm";
```
Comment (jonkoops) [inline js/apps/admin-ui/cypress/support/pages/admin-ui/CreateRealmPage.ts:4]: This should be replaced by a `data-testid`

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Review by jonkoops — APPROVED** [review]
LGTM

**Pull request merged**